shared/models/template_deletion.yml

```diff
@@ -5,7 +5,8 @@ description: >-
 
```

Comment:
Do you still need this file?

Reply:
I don't! I'm going to delete every iteration of this file across the resources